lib/controllers/favorites_controller.js

```diff
@@ -11,6 +11,21 @@ const index = (request, response) => {
     })
 }
 
+const show = (request, response) => {
+  Favorite.find(request.params.id)
+    .then((favorite) => {
+      if (typeof favorite !== "undefined") {
+        response.status(200).json(favorite);
+      } else {
+        response.status(404).json("There are no Favorites with that ID")
+      }
```

Comment:
This can be cleaned up a bit. Since `Favorite::find()` returns either an object or undefined, you don't need to check to see if the return value is "undefined"

This means you can do something like the following
```
if (favorite)
  status(200)
else
  status(404)
```

It can be cleaned up more if we change `response, request` to `res, req` and allow us to remove the if-statement like so:
```javascript
.then(favorite => {
  favorite ? res.status(200).json(favorite) 
           : res.status(404).json("There are no Favorites with that ID")
}
```

Though, this could be close to the 80 character limit.

Reply:
Great points!

Reply:
I implemented your logic